src/components/forms/file-dropper/file-dropper.tsx

```diff
@@ -45,11 +47,24 @@ interface FileDropperProps {
   /**
    * (Optional) Categories that the user can add to each uploaded file (e.g. 'CV' or 'Cover Letter')
    * */
-  fileCategories?: string[];
+  fileCategories?: { value: string; label: string }[];
   /**
    * (Optional) Pass in a valid illustration keyword to show the according illustration.
    */
   illustration?: IllustrationType;
+  /**
+   * (Optional) label, which is displayed above
```

Comment:
`(Optional)` is not needed. This information is already in the type itself (`...?`)

Reply:
I am still learning to stop doing this 😅 

![image](https://user-images.githubusercontent.com/78978542/160349646-ffc7296a-5936-40c9-8d91-924182d4a51f.png)